hypothesis-python/RELEASE.rst

```diff
@@ -0,0 +1,7 @@
+RELEASE_TYPE: patch
+
+Tidied up internal defines_strategy decorators code. :issue:`#2516`
+
+The defines_strategy decorator now has optional named parameters instead of using a set of decorators with
+different options "preset".
+
```

Comment:
```suggestion
This patch contains some internal refactoring.  Thanks to Felix Sheldon for fixing :issue:`#2516`!
```

Because it's an internal-only change, we don't discuss the details in the changelog - but do want to thank our contributors :wink: